nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -297,6 +297,11 @@
      <literal>phd</literal> service have been removed from nixpkgs due to lack of maintainer.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The <literal>mkOverrideUnset</literal> function and <literal>mkUnset</literal> constant have been added to <literal>lib</literal>.
```

Comment:
In `option-def.xml` the `mkOverrideUnset` and `mkUnset` names were inside `varname` elements. Might be nice to continue that here?

Reply:
Okay, done!